### ISSUE TITLE
returning lit specific object types

### ISSUE THREAD

**Issue by navybluesilver** [mit-dci/lit-rpc-client-go#3]
Can we return the arrays / lists in json format to provide better orthogonality? Introducing "lit specific" objects forces the users to import libraries other than the "lit-rpc-client-go", which is not desirable.

**Comment by gertjaap** [comment]
I agree, we can probably copy the originals into a local models namespace. This is the case in the other language clients already.

**Comment by navybluesilver** [comment]
Perhaps a long term solution could be protobuf/gRPC. This will also help to support other languages (including Node.js and c#). LND has something similar: https://github.com/lightningnetwork/lnd/tree/master/lnrpc
